### ISSUE TITLE
Switch back to letting rhc create the local git clone?

### ISSUE THREAD

**Issue by ncoghlan** [ncoghlan/openshift-kallithea#9]
@GrahamDumpleton, in your redesign, you switched to cloning the quickstart repo, and then adding the openshift repo as a custom remote, rather than the other way around.

I'm inclined to change it back, but didn't want to do that before asking for more info on why you decided to change it in the first place.

**Comment by GrahamDumpleton** [comment]
You probably know what works better for OpenShift at that point. I am also not a git expert but I find what I switched it to a more logical way of doing it since OpenShift isn't the primary home for my code but just a place to push for deployment. Guess I have been brainwashed by the heroku approach.

**Comment by ncoghlan** [comment]
Yeah, I guess it depends on if you're viewing it as just a "deployment
instructions repo" or the main app repo. For Kallithea, I think it's more
on the "deployment instructions" side of things.
